Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -550,7 +588,7 @@
     "    json_request = f'{{ \"instances\" : {data_formatted} }}'\n",
     "\n",
     "    # Specify the prediction URL. If you are runing this notebook outside of Kubernetes cluster, you should set the Cluster IP.\n",
-    "    url = f\"http://{name}-predictor-default.{namespace}.svc.cluster.local/v1/models/{name}:predict\"\n",
+    "    url = f\"http://{name}-predictor.{namespace}.svc.cluster.local/v1/models/{name}:predict\"\n",
```

Comment:
what is value of {name} var?

Reply:
It's the name of the katib-experiment. It is declared in the 2nd cell. 

![image](https://github.com/HPEEzmeral/ezua-tutorials/assets/99253324/75bc3518-4b9b-4730-a85a-c09c9007df44)